### ISSUE TITLE
CVE-2022-1292 patched?

### ISSUE THREAD

**Issue by floek** [bitnami/bitnami-docker-nginx-ingress-controller#20]
### Name and Version

bitnami/nginx-ingress-controller:1.2.1-debian-11-r14

### What steps will reproduce the bug?

Install bitnami/nginx-ingress-controller:1.2.1-debian-11-r14

### What is the expected behavior?

Up to date openssl

### What do you see instead?

```
$ openssl version
OpenSSL 1.1.1n  15 Mar 2022
```

### Additional information

Can you please state, if openssl Remote Code Execution bug (CVE-2022-1292) is fixed in your latest image?

**Comment by rafariossaa** [comment]
Hi,
  According to the information provided by the debian CVE tracker it was fixed in openssl version `1.1.1n-0+deb11u3`.
  You can check the version delivered in the container by running:
```
$ docker run --entrypoint bash -it bitnami/nginx-ingress-controller:1.2.1-debian-11-r14 

I have no name!@ec3d6aabe80f:/opt/bitnami/nginx-ingress-controller$ dpkg -l | grep -i openssl
ii  libcurl4:amd64             7.74.0-1.3+deb11u1             amd64        easy-to-use client-side URL transfer library (OpenSSL flavour)
ii  libcurl4-openssl-dev:amd64 7.74.0-1.3+deb11u1             amd64        development files and documentation for libcurl (OpenSSL flavour)
ii  openssl                    1.1.1n-0+deb11u3               amd64        Secure Sockets Layer toolkit - cryptographic utility
```

**Comment by floek** [comment]
Thank you! The updated version is installed.
